internal/runbits/refresh.go

```diff
@@ -0,0 +1,40 @@
+package runbits
+
+import (
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/pkg/platform/runtime"
+	"github.com/ActiveState/cli/pkg/project"
+	"github.com/go-openapi/strfmt"
+)
+
+// RefreshRuntime should be called after runtime mutations. A nil arg for "req"
+// means that the message handler will not print output for "a single
+// requirement". For example, if multiple requirements are affected, nil is the
+// appropriate value.
+func RefreshRuntime(out output.Outputer, req *RequestedRequirement, proj *project.Project, cachePath string, commitID strfmt.UUID, changed bool) error {
```

Comment:
Feels a little awkward here, filed this followup: https://www.pivotaltracker.com/story/show/176930119